ISSUE_TEMPLATE/component_validation_checklist.md

```diff
@@ -0,0 +1,61 @@
+# Component validation checklist
+
+Here is a checklist of items that need to be completed before a component or pattern can be marked as Ready in the Design System.
+
+- [ ] Documentation is complete. [Documentation writing info](#)
+    - Read through documentation and make sure the following is done
+      - [ ] Description of component or pattern
+      - [ ] HTML code snippet example(s)
+      - [ ] React code snippet example(s)
+      - [ ] All react properties are explained
+      - [ ] Guidance on when to use this item
+      - [ ] Guidance on when not to use this item
+      - [ ] Accessibility considerations
+
+---
+
+- [ ] Validate HTML
+  - Go to [validator.w3.org](https://validator.w3.org/#validate_by_input) paste your code in and make sure it validates without error.
+
+---
+
+- [ ] Keyboard testing
+  - Navigate the item using your Keyboard
```

Comment:
What do you think about making this a link to the WebAIM page on keyboard navigation? https://webaim.org/techniques/keyboard/